src/components/Dashboard/ProprietorPartnersListSm.vue

```diff
@@ -147,11 +147,12 @@ $avatar-width: 2.75rem;
 
 // Complete filing required styling
 .complete-filing {
-  padding: 2rem;
-  color: $gray6;
+  padding: 1.2rem;
```

Comment:
Don't do this. It's going away from our standardized spacing in this UI.

Reply:
I was discussing with tracey, to fix alignment, May be i can leave it as it is 
<img width="766" alt="Screen Shot 2022-05-30 at 2 19 14 PM" src="https://user-images.githubusercontent.com/55506249/171060119-2eeebd7b-6bee-4749-9e61-33f03dac23ea.png">

Reply:
The alignmnent is off because the component expects to display an icon for the person's name. There is already an if-then-else in the code to display the icon+name or "not entered". I think it just needs to be adjusted.